app/controllers/salt_controller.rb

```diff
@@ -0,0 +1,29 @@
+require "velum/salt"
+require "velum/salt_api"
+# HealthController holds methods for checking the readiness of the application.
+class SaltController < ApplicationController
+	include Velum::SaltApi
+ 	skip_before_action :redirect_to_setup
+
+	def update
+
+    minions = perform_request(endpoint: "/minions", method: "get")
```

Comment:
This can be removed using `Salt.minions` that is already implemented: https://github.com/kubic-project/velum/blob/master/lib/velum/salt.rb#L24